Reportedly/Utilities/LocalNotificationScheduler.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LocalNotificationScheduler.swift
+//  Reportedly
+//
+//  Created by Minh Pham on 09/12/2020.
+//
+
+import Foundation
+import UIKit
+import UserNotifications
+
+final class LocalNotificationScheduler {
+    
+    static let shared = LocalNotificationScheduler()
+    
+    func setupDailyStandupLocalNotifications() {
+        // Create the notification content
+        let content = UNMutableNotificationContent()
+        content.title = "Daily Standup Report"
+        content.body = "Hi, it's time to start the daily standup. Please, open the app and answer your daily questions"
```

Comment:
Should we use localization for title and body here?  😅

Reply:
oh I forget about it, let me update 😭